### DISCUSSION TITLE
Visual Studio intellisense

### DISCUSSION THREAD

**Discussion by ignatandrei** [dotnet/csharplang#4314] [Q&A] [answered]
Hello
I have created a Roslyn Code Generator - makes actions in controllers from an attribute ( see https://www.nuget.org/packages/SkinnyControllersGenerator/  )

However, from Visual Studio, I cannot see the actions generated, even if there are generating and executing

What should I do to provide intellisense suppoert in VS?

Thanks
Andrei

**Comment by YairHalberstadt** [comment, marked as answer]
Hi

Please open a discussion on dotnet/Roslyn for this, or discuss on gitter.

Thanks
